src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/AuthorizationFunctionalTest.java

```diff
@@ -399,6 +399,34 @@ public void validateRetrievedUsers(Map<String, Object> searchResponse, String ex
         });
     }
 
+    public void validateRetrievedUsersDetails(Map<String, Object> searchResponse) {
+        assertThat(searchResponse.get("users")).asList().isNotEmpty();
+        assertThat(searchResponse.get("lastRecordInPage")).isNotNull();
+        assertThat(searchResponse.get("moreAvailable")).isNotNull();
+        List<HashMap> professionalUsersResponses = (List<HashMap>) searchResponse.get("users");
+
+        professionalUsersResponses.forEach(user -> {
+            assertThat(user.get("userIdentifier")).isNotNull();
+            assertThat(user.get("lastUpdated")).isNotNull();
+
+            HashMap<String, String> orgInfo = (HashMap<String, String>) user.get("organisationInfo");
+            assertThat(orgInfo).isNotNull();
+            assertThat(orgInfo.get("organisationIdentifier")).isNotNull();
+            assertThat(orgInfo.get("status")).isNotNull();
+            assertThat(user.get("lastUpdated")).isNotNull();
+
```

Comment:
check expected value for not null fields

Reply:
Thank you for review. Raj, its common methods and generate dynamic value for multiple value so not possible to assert exact value. I have added assert for status.